debian/control

```diff
@@ -19,10 +19,10 @@ Build-Depends:
  libglib2.0-dev (>= 2.50.0),
  libgsystem-dev,
  libnm-dev (>= 1.2.0),
- libostree-dev (>= 2016.15+dev24.a109440),
+ libostree-dev (>= 2017.12),
  libsoup2.4-dev,
  libsystemd-dev,
- ostree,
+ ostree (>= 2017.12),
  ostree-tests (>= 2017.6),
```

Comment:
No, we don’t use anything from the new version of that package which isn’t in the old version. In practice, we will always get `ostree-tests (>= 2017.12)` because the `debian/control` file for the `ostree` package versions them in lockstep.